module-apps/apps-common/popups/lock-popups/SimSwitchingWindow.cpp

```diff
@@ -46,7 +46,7 @@ namespace gui
                     case Store::GSM::SIM::SIM_FAIL:
                     case Store::GSM::SIM::SIM_UNKNOWN:
                         if (std::chrono::steady_clock::now() - startTime >= constants::maxDurationTime) {
-                            LOG_INFO("The SIM switch operation failed. We return to the previous window.");
+                            LOG_WARN("The SIM switch operation failed. We return to the previous window.");
```

Comment:
I'm not sure whether the part about returning to previous window gives any useful information